source/Octopus.Tentacle.Contracts/ScriptServiceV3Alpha/KubernetesAgentScriptExecutionContext.cs

```diff
@@ -2,18 +2,18 @@
 
 namespace Octopus.Tentacle.Contracts.ScriptServiceV3Alpha
 {
-    public class KubernetesJobScriptExecutionContext : IScriptExecutionContext
+    public class KubernetesAgentScriptExecutionContext : IScriptExecutionContext
     {
         [JsonConstructor]
-        public KubernetesJobScriptExecutionContext(string image, string? feedUrl, string? feedUsername, string? feedPassword)
+        public KubernetesAgentScriptExecutionContext(string image, string? feedUrl, string? feedUsername, string? feedPassword)
         {
             Image = image;
             FeedUrl = feedUrl;
             FeedUsername = feedUsername;
             FeedPassword = feedPassword;
         }
 
-        public KubernetesJobScriptExecutionContext()
+        public KubernetesAgentScriptExecutionContext()
```

Comment:
I ended up choosing to name this `KubernetesAgent` because I felt it was specific enough to how the agent works